edumanage/models.py

```diff
@@ -100,30 +101,71 @@ def value_to_string(self, obj):
         value = self._get_val_from_obj(obj)
         return self.get_db_prep_value(value)
 
-ERTYPES = (
-    (1, 'IdP only'),
-    (2, 'SP only'),
-    (3, 'IdP and SP'),
+# https://www.djangosnippets.org/snippets/2402/
+def get_namedtuple_choices(*choices_tuples):
+    names = [tup[0] for tup in choices_tuples]
+    vals = [tup[1:3] for tup in choices_tuples]
+    class Choices(namedtuple('Choices', names)):
+        def __getattribute__(self, name):
+            attr = super(Choices, self).__getattribute__(name)
+            if not isinstance(attr, tuple):
+                return attr
+            return attr[0]
+    return Choices._make(vals)
+
+_ERTYPES = (
+    ('IDP', 1, 'IdP only', 'IdP'),
+    ('SP', 2, 'SP only', 'SP'),
+    ('IDPSP', 3, 'IdP and SP', 'IdP+SP'),
+)
+ERTYPES = get_namedtuple_choices(*_ERTYPES)
+def get_ertype_string(ertype):
```

Comment:
This function should document the input parameter.  I assume it is the numeric ID, but it should be documented.

Reply:
Alright